modules/policy_document/main.tf

```diff
@@ -19,73 +19,73 @@ locals {
   ]
 }
 
-data aws_iam_policy_document empty {}
+data "aws_iam_policy_document" "empty" {}
```

Comment:
Regarding line 12:  How do you ensure that you will have no more than 10 documents?   Or what happens if you have more than 10 documents?  Do they just override the "aws_iam_policy_document ten"?

Reply:
Very good question. It is a limitation of this implementation that `template_paths` only supports up to 10 values. We test for that in the variable definition using the variable validation feature that is new in tf 0.13, here: https://github.com/plus3it/terraform-aws-tardigrade-iam-principals/blob/1c7e59e2a9f5dbb8c4f21a9dfb4bd5459034cf13/modules/policy_document/variables.tf#L6-L13

Reply:
Missed that and I started with policy_document.